Goal: Adapt example script to Debian

Fixes: #346234

Status wrt upstream: Debian specific

Note: 

Index: samba-3.0.25c/docs/manpages/smb.conf.5
===================================================================
--- samba-3.0.25c.orig/docs/manpages/smb.conf.5	2007-08-26 12:07:15.098370040 +0200
+++ samba-3.0.25c/docs/manpages/smb.conf.5	2007-08-26 13:09:05.419555107 +0200
@@ -662,8 +662,8 @@
 Default:
 \fB\fIadd machine script\fR = \fR
 .sp
-Example:
-\fB\fIadd machine script\fR = /usr/sbin/adduser -n -g machines -c Machine -d /var/lib/nobody -s /bin/false %u \fR
+Example for Debian:
+\fB\fIadd machine script\fR = /usr/sbin/adduser -n -g machines -c Machine -d /var/lib/samba -s /bin/false %u \fR
 .RE
 .PP
 add port command (G)
